tests/conftest.py

```diff
@@ -23,3 +23,50 @@ def es(docker_services):
 @pytest.fixture(scope='session')
 def es_client(es):
     return Elasticsearch([es])
+
+
+@pytest.fixture(scope='session')
+def wiki_es(docker_services):
+    """Ensure that ES is up and responsive."""
+    docker_services.start('wiki_es')
+    port = docker_services.wait_for_service("wiki_es", 9200)
+
+    url = f'http://{docker_services.docker_ip}:{port}'
+
+    for c in app.injector.components:
+        if isinstance(c, SettingsComponent):
+            c._settings['WIKI_ES'] = url
+            c._settings['ES_WIKI_LANG'] = 'fr'
+    return url
+
+
+@pytest.fixture(scope='session')
+def wiki_es_client(wiki_es):
+    return Elasticsearch([wiki_es])
+
+
+@pytest.fixture(scope="session")
+def init_indices(es_client, wiki_es_client):
```

Comment:
now that we have another es_client it might be nice to rename `es_client` to `mimir_client` ?

Reply:
done